mica/stats/acq_stats.py

```diff
@@ -441,6 +448,43 @@ def _get_obsids_to_update(check_missing=False):
     return obsids
 
 
+def warn_on_acq_anom(acq, starcheck, to_addr):
+    BOX_PAD = 16
+    anom_match = (((np.abs(acq['yang_obs'] - acq['yang']) > (acq['halfw'] + BOX_PAD))
+                   | (np.abs(acq['zang_obs'] - acq['zang']) > (acq['halfw'] + BOX_PAD))))
+    if not np.any(anom_match):
+        return
+    for anom_idx in np.flatnonzero(anom_match):
+        anom = acq[anom_idx]
+        if anom['img_func'] == "NONE":
+            # if there was nothing tracked, there is really isn't an anomaly
+            continue
+        other_box_match =  ((np.abs(acq['yang'] - anom['yang_obs']) < (acq['halfw'] + BOX_PAD))
+                            & (np.abs(acq['zang'] - anom['zang_obs']) < (acq['halfw'] + BOX_PAD)))
+        if np.any(other_box_match):
+            text = "Acquisition Anomaly.  Star for slot {} actually in box {} \n".format(
+                anom['slot'], acq[other_box_match][0]['slot'])
+        else:
+            text = "Does not appear to be classic star-in-wrong-box anomaly\n"
+        text += """Large Deviation from Expected ACQ Star Position in {obsid}
+      Slot {slot} Expected (Y-Pos, Z-Pos) = ({yang:.1f}, {zang:.1f})
+      Slot {slot} Observed (Y-Pos, Z-Pos) = ({yang_obs:.1f}, {zang_obs:.1f})
+      Halfwidth {halfw:03d}        (dy, dz) = ({dy:.1f}, {dz:.1f})""".format(
+            **{'obsid': anom['obsid'], 'yang': anom['yang'], 'zang': anom['zang'],
```

Comment:
For this I was trying to remember if Table.Row had an option to give me a dict() without doing it manually.